chat-plugins/uno.js

```diff
@@ -475,6 +478,7 @@ exports.commands = {
 		timer: function (target, room, user) {
 			if (!this.can("minigame", null, room)) return;
 			if (!room.game || room.game.gameid !== "uno") return this.errorReply("There is no UNO game going on in this room.");
+			target = target.replace(/[0-9]/g, '');
```

Comment:
What you want it to do is bad, anyway. Failing silently was old PS behavior, new PS behavior is to actually give error messages so we don't have unknown bugs.